src/server/transaction.h

```diff
@@ -278,9 +282,12 @@ class Transaction {
   // scheduled transaction is accessed between operations as well.
   absl::InlinedVector<PerShardData, 4> shard_data_;  // length = shard_count
 
-  //! Stores arguments of the transaction (i.e. keys + values) partitioned by shards.
+  // Stores arguments of the transaction (i.e. keys + values) partitioned by shards.
   absl::InlinedVector<std::string_view, 4> args_;
 
+  // Stores the full command.
+  CmdArgList full_args_;
```

Comment:
I usually go with the approach of giving more specific variable/members names and not use comment.
Here it can be cmd_with_args_

Reply:
I called it `cmd_with_full_args_`, but kept a comment is its more clear